### ISSUE TITLE
Can tags point to multiple projects/commits?

### ISSUE THREAD

**Issue by aalten** [githubteacher/advanced-apr-2015#13]

**Comment by pcharlan** [comment]
No; a tag points to one thing.  It can contain an arbitrary textual message, but Git associates the tag with only one SHA1 (which is usually the SHA1 of a commit, but could be a tree or blob or (I think) another tag).

**Comment by aalten** [comment]
thanks
